_includes/expand-button.html

```diff
@@ -0,0 +1 @@
+<button style="width: max-content" class="usa-button desktop:margin-top-3 margin-bottom-2 bg-primary-darker" id="expand-all">Expand all</button>
```

Comment:
cc @nguyen-tina, but I would change this button color to match the headers, something like:

![image](https://github.com/usdoj-crt/beta-ada/assets/15126660/1e8b5430-7abc-445d-a7d2-12786f6d97d4)

Reply:
The button matches the print button below it right now so unless we're going to change the color of that I think it might look strange to have it a slightly different blue.

Reply:
I think the `+` is `#000` right now, and the print button is `#172E51`?

![black](https://github.com/usdoj-crt/beta-ada/assets/15126660/85fcf69c-53fb-4951-a9a6-45838c7b4b10)

Reply:
oh you're talking about the +s? I'm not sure why those are black but I didn't touch those as part of this change so I'm inclined to leave them as is

Reply:
I'm with Katie on that – I don't think the +'s are part of this, and am also inclined to leave them as-is

Reply:
sgtm!